interfaces/acados_template/acados_template/acados_ocp_solver.py

```diff
@@ -195,7 +195,15 @@ def create_cython_solver(cls, json_file):
     def save_p_global(self) -> bool:
         return self.__save_p_global
 
-    def __init__(self, acados_ocp: Union[AcadosOcp, AcadosMultiphaseOcp], json_file=None, simulink_opts=None, build=True, generate=True, cmake_builder: CMakeBuilder = None, verbose=True, save_p_global=False):
+    @property
+    def N(self) -> int:
```

Comment:
N_horizon?

Reply:
this field was already there in Python, just not as a property.
It is also used in an example.
We can rename it, but it is breaking at this point.